14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,96 @@
+
+The Swagger documentation allows you to select a definition and go to either Umbraco Management API or Content Delivery API. If you are extending the Management API with your own controllers, you can also create custom documentation for these in Swagger. See [Custom Swagger API](../custom-swagger-api.md) article for details.
+
+![Umbraco Management API documentation in Swagger](../images/management-api-swagger.png)
+
+In the Swagger Umbraco Management API, you can find a collection of available endpoints in this version of Umbraco.
+
+## Authorization
+
+The Management API endpoints are protected by the backoffice authorization policy and need an authentication token to interact with them.
+
+To set the authorization, click on the **Authorize** button:
+
+![Umbraco Management API Authorize Button](../images/management-api-swagger-authorize-button.png)
+
+Then a popup will appear with some setup information and login form for authorization:
+
+![Umbraco Management API Authorize Login](../images/management-api-swagger-authorize-instructions.png)
+
+The available integration for the authorization is done via a backoffice user with the integration of `OAuth2, authorizationCode with PKCE`. Swagger is only enabled in non-production environments, so if you need to access the Management API in production, you need a different client.
+
+{% hint style="info" %}
+
+In production environment, only `umbraco-back-office` **client** is allowed to connect to the Management API. In non-production environments, the `umbraco-swagger` and `umbraco-postman` **clients** can be used.
+
+You can see an example of how to connect a backoffice user via OAuth2 in Postman in the [Swagger Setup in Postman](./postman-setup-swagger) article.
+
+{% endhint %}
+
+When in non-production environment, you will need to connect via OAuth2 with the following Umbraco Authentication specifications:
+
+**Authorization URL**
+
+```http
+{yourdomain}/umbraco/management/api/v1/security/back-office/authorize
+```
+
+**Token URL**
+
+```http
+{yourdomain}/umbraco/management/api/v1/security/back-office/token
+```
+
+**Flow**
+
+```http
+authorizationCode with PKCE
+```
+
+**client_id**
+
+```http
+umbraco-swagger
+```
+
+# Test an Endpoint
+
+To test a Management API endpoint, follow these steps:
+
+1. Authenticate via the **Authorize** button. Make sure you use `umbraco-swagger` as the `client_id`:
+
+![Umbraco Management API when Authenticated](../images/management-api-swagger-authenticated.png)
+
+{% hint style="info" %}
+In non-production environments, you will always need only the `client_id` to authenticate. The `client_secret` should always be left blank because of the chosen authentication flow.
```

Comment:
```suggestion
In non-production environments, you will always need only the `client_id` to authenticate. The `client_secret` should always be left blank as no secrets have been configured for the development-only clients. 
```

Reply:
I would go with something like "...should always be left blank, as the authentication flow does not use client secret."

Reply:
thank you for your suggestion, I have applied it 🙌